tests/comparisons/Model/testBakeEnum.php

```diff
@@ -0,0 +1,18 @@
+<?php
+declare(strict_types=1);
+
+namespace Bake\Test\App\Model\Enum;
+
+/**
+ * FooBar Enum
+ */
+enum FooBar: int
+{
+    /**
+     * @return string
+     */
+    public function label(): string
+    {
+        return mb_strtolower($this->name);
```

Comment:
"Humanized" inflection would be better here as these are the values displayed to the user.

Reply:
This also implies a utf8 name. Is that something that exists?

Reply:
Entertainingly the following is valid PHP

```php
<?php

enum Foo : string {
    case FIRST = 'first';
    case 😃 = 'unicode';
}
echo Foo::FIRST->value;
echo Foo::😃->value;
```